components/toolbar/built-in-tools.md

```diff
@@ -102,7 +102,7 @@ The nested `ToolBarToggleButton` tag exposes parameters that allow you to custom
 | `Selected` | `bool` | Specifies whether the button is in selected state. You can use it with one and two-way data binding with the `SelectedChanged` event. For more information on how to handle the `SelectedChanged` event see the [Events]({%slug toolbar-events%}) article. |
 | `SpriteClass` | `string` | Add a sprite class image to the button. Set this attribute to `k-icon MySpriteClass` where `MySpriteClass` defines the CSS rules for the sprite. |
 | `Overflow` | `ToolBarItemOverflow` enum <br /> (`Auto`) | Specifies whether the item will be hidden when the ToolBar is resized. |
-| `OverflowText` | `string` | If not specified, it will get the normal text of the item. (not valid for Separators) |
+| `OverflowText` | `string` | Defines the buttons text that will be shown only in the overflow popup. If not specified, it will get the default text of the item. (not valid for Separators) |
```

Comment:
```suggestion
| `OverflowText` | `string` | Defines the button text that will be shown only in the overflow popup. If not specified, it will get the default text of the item. (not valid for Separators) |
```